### PR TITLE
Update for callout name change

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Websites has updated the beta-callout to just callout.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by hestonhoffman — APPROVED** [review]
LGTM!

File: snippets/markdown.json [hunk 1]
```diff
@@ -69,14 +69,14 @@
 		"description": "Alert box with choice of info, danger, or warning"
 	},
 
-	"Beta callout":{
-		"prefix": ";;beta",
+	"Callout":{
+		"prefix": ";;callout",
 		"body": [
-			"{{< beta-callout url=\"https://google.com\" d-toggle=\"modal\" d_target=\"#signupModal\" custom_class=\"sign-up-trigger\" btn_hidden=\"false\">}}",
+			"{{< callout url=\"https://google.com\" d-toggle=\"modal\" d_target=\"#signupModal\" custom_class=\"sign-up-trigger\" btn_hidden=\"false\">}}",
```
Comment (StefonSimmons) [inline snippets/markdown.json:75]: `d-toggle=\"modal\"` should be `d_toggle` with an underscore

Comment (maycmlee) [inline snippets/markdown.json:75]:
```suggestion
			"{{< callout url=\"https://google.com\" d_toggle=\"modal\" d_target=\"#signupModal\" custom_class=\"sign-up-trigger\" btn_hidden=\"false\">}}",
```

Comment (maycmlee) [reply snippets/markdown.json]: Thanks @StefonSimmons, good catch!

**Pull request merged**